rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date)
+    @date = date
+    @person = person
+    @book = book
+    person.rental << self
+    book.rental << self
```

Comment:
- [ ] There should be `person` and `book` parameters in your **`initialize`** method, for your respective instance variables to work. Kindly add them.

  Remember to ensure your `Rental.new` return statements in the `add_rental` methods in both **Person** and **Book** classes follow the same declarations.